docs/src/pages/Conformance.md

```diff
@@ -21,6 +21,54 @@ The HL7 FHIR specification is more than just a data format. It defines an [HTTP
 
 The IBM FHIR Server implements a linear versioning scheme for resources and fully implements the `vread` and `history` interactions, as well as version-aware updates.
 
+### Configuring allowable API interactions
+By default, the IBM FHIR Server allows all supported API interactions (`create`, `read`, `vread`, `history`, `search`, `update`, `patch`, `delete`). However, it is possible to configure which of these interactions are allowed on a per resource basis through a set of interaction rules specified via the `fhirServer/resources/<resourceType>/interactions` property in `fhir-server-config.json`. The following snippet shows the general form for specifying interaction rules:
```

Comment:
I very much like having this section here, but in my opinion I think we should:
1. keep it high-level and more client-focused here
2. move the "how to configure it in fhir-server-config.json" stuff into the user's guide

Reply:
Maybe this section just becomes a paragraph that says its configurable and then links to the user's guide for the detail?

Reply:
That makes sense. I was waffling between putting it in the conformance guide and the user's guide. Wasn't quite sure where it would go in the user's guide. Do you think a new sub-section in Chap.4? Maybe titled "FHIR HTTP API" or "FHIR REST API"?

Reply:
Discussed with Lee - will add a new section in Chap. 4.

Reply:
Updated doc looks good to me